net/prosody/Makefile

```diff
@@ -44,7 +45,7 @@ define Package/prosody/conffiles
 /etc/prosody/prosody.cfg.lua
 endef
 
-TARGET_CFLAGS += $(FPIC) -std=gnu99
```

Comment:
Why it was removed?

Reply:
GCC starting with 5 defaults to the highest stable version.